packages/ui-kit-core/README.md

```diff
@@ -0,0 +1,11 @@
+# `ui-kit-core`
```

Comment:
I would try to find another name, core seems a little unspecific, like a kitchen sink

Reply:
I am open for suggestions and happy to change this. This is basically the package which contains all the packages.
`ui-kit-general`. I did not want to go with `ui-kit` since it basically is a different package then `ui-kit`. I just picked something which kinda made sense and I choose not to be blocked on something which ain't important right now.